server/spec/requests/api/v1/wishes_spec.rb

```diff
@@ -24,20 +27,64 @@
             expect(json.size).to eq(5)
           end
         end
-        response '404', 'interval not found' do
+        response '404', 'User not found' do
           let(:api_user_id) { 'invalid' }
           run_test! do
             expect(json['message']).to match(/Couldn't find ApiUser with 'id'=#{api_user_id}/)
           end
         end
       end
+    end
+
+    path '/api/v1/api_users/{api_user_id}/wishes/{id}' do
+
+      get 'Displays a Wish for a particular user' do
+        tags 'List a Wish for a particular user'
+        consumes 'application/json'
+        parameter name: :api_user_id, :in => :path, :type => :string
+        parameter name: :id, :in => :path, :type => :string
+
+        response '200', 'list wish' do
+          run_test! do
+            expect(json["goal"]).not_to eq(nil)
+          end
+        end
+        response '404', 'Wish not found' do
+          let(:id) { 'invalid' }
+          run_test! do
+            # expect(json['message']).to match(/Couldn't find Wish with 'id'=#{id}/)
```

Comment:
Does this test lack an expectation?

If the commented-out code is not doing anything, it can be removed.

It seems the User test _is_ checking for the presence of a validation error in the 'message' key of the response.

Reply:
The object referenced in the error message produced looks strange, I've created an issue to figure out what is happening with. At the moment the test just checks that a 404 is error is produced with an invalid ID

Reply:
Issue #278